### ISSUE TITLE
A few small enhancements

### ISSUE THREAD

**Issue by 5183nischal** [intellibar/main#36]
Hi, I am really enjoying the app and it's recent updates. Here are a few things that I think would be nice for me personally:
1.  The same shortcut that opens 'Ask' also closes it.
	Currently, the chose shortcut open the Ask window on side but then you need to press Esc to close it. I think the command toggling it on and off might be nice.
2. Rounded edges. 
Mac has rounded corners for everything, so the sharp corners looks a little jarring.
3. Save chats locally. 
It would be great to be able to specify the folder where the chat logs are stored. Maybe it already exists and I just don'tknow how to use it.

Thank you!

**Comment by erusev** [comment]
> It would be great to be able to specify the folder where the chat logs are stored. Maybe it already exists and I just don'tknow how to use it.

What's the use case?

Thanks!

**Comment by 5183nischal** [comment]
Mostly because I have chat logs coming from other sources (for example talking to LLMs from inside my editor) and it would be great to have a centralized location where all my conversations across all apps are stored. It's nice to have but not super crucial. Thank you!

**Comment by erusev** [comment]
Thanks! We'll consider it.

**Comment by rb81** [comment]
I agree with @5183nischal. It would be good to select a folder, for transparency if nothing else, but also serves a practical use-case for many. I would also suggest easy import and export of chats.
